samples/bluetooth/mesh/light_ctrl/README.rst

```diff
@@ -122,6 +123,10 @@ Configuration
 
 |nrf5340_mesh_sample_note|
 
+The Kconfig option :kconfig:option:`CONFIG_BT_MESH_LIGHT_CTRL_REG_SPEC` is set by default as it is necessary for the :ref:`bt_mesh_light_ctrl_srv_readme` model according to the `Bluetooth mesh model specification`_.
+	The option enables a separate module called illuminance regulator.
+	For more information about the module, see the documentation on :ref:`bt_mesh_light_ctrl_reg_readme` and :ref:`bt_mesh_light_ctrl_reg_spec_readme`.
```

Comment:
It seems like there are some indenting issues for the two lines. Not sure what happened.